### DISCUSSION TITLE
Night-Shift for lightmode

### DISCUSSION THREAD

**Discussion by KripC2160** [Orama-Interactive/Pixelorama#463] [Ideas]
All though I am not a light mode user, I think the use of the Night-shift feature can be cool. It seems to be a feature that can be implemented pretty easily as well. I am thinking of something like a button appearing in the editor when you choose the light mode and interpolate between nightshift on and off. That's all I have to say

**Comment by novhack** [comment]
Do you mean that there should be a feature that yellows the light UI according to current daytime?
If yes I think it's not needed. Most people that want this feature want it for all of their apps so they set it up in their OS settings. All operating systems supported by Pixelorama can apply the tint globally.
